force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -94,8 +123,19 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the input is "Empty", "Null", or "Contains Text".
      */
     public static String checkString(String a) {
-        return null; // Replace null with the variable you used to store the result
+             
+       if(a == null){
+         return 'Null';
+       
+        } else if (!String.isEmpty(a)) {
```

Comment:
Good use of string method. This will also be true is the string is null, but you checked for null specifically before this, so this is ok.